packages/bvaughn-architecture-demo/pages/variables.css

```diff
@@ -102,6 +107,11 @@
   --color-secondary-button: #ffffff;
   --color-warning: #6c5914;
 
+  --color-primary-background: #ffffff;
```

Comment:
nit
```suggestion
  --color-primary-background: var(--background-color-contrast-1);
```

Reply:
👍 I'll be in here soon for more cleanup. I want to do a full pass on colors and get things sorted, probably after icon cleanup.